packages/lwc-language-server/src/lwc-server.ts

```diff
@@ -454,7 +463,7 @@ export default class Server {
                 const match = findDynamicContent(content, relativeOffset);
 
                 if (match) {
-                    const item = iterators.find(i => i.name === match) || null;
+                    const item = iterators.find((i) => i.name === match) || null;
```

Comment:
I know this is pretty far behind, eslint-wise, but prefer `??` over `||`. 

It doesn't matter until it does (falsy 0, etc) so it's a good rule/habit.